### DISCUSSION TITLE
为其他脚本的类添加夜间模式支持

### DISCUSSION THREAD

**Discussion by peach0x33a** [the1812/Bilibili-Evolved#2445] [功能建议 / Ideas]
[翻页评论区脚本](https://greasyfork.org/zh-CN/scripts/403348)
![image](https://user-images.githubusercontent.com/79032826/139561938-37ee8101-c1d6-4da3-8e00-83e62752edee.png)
(类：dialog btn-hover)
![image](https://user-images.githubusercontent.com/79032826/139561982-e195028d-4762-4346-b722-5c920a915b47.png)
(类：comment-dialog-container)

[枝江网络查重](https://greasyfork.org/zh-CN/scripts/429068)
![image](https://user-images.githubusercontent.com/79032826/139562018-819965e6-19da-4165-b17e-bcc670c98d0a.png)
(类：asoulcnki btn-hover btn-highlight)

**Comment by the1812** [comment]
建议自己弄个 stylus 样式上去 (以后还能放在 v2 里用), 放夜间里对于其他不用这些脚本的人毫无用处.

**Reply by peach0x33a** [reply]
好的

**Comment by Geeyun-JY3** [comment]
翻页评论区这个脚本的这个查看对话好像是 B 站还在测试的功能，以后应该也是要适配的：https://github.com/the1812/Bilibili-Evolved/issues/2136#issuecomment-922230263
另外这个脚本本质上就是还原以前的旧版评论区，所以只要不把以前对旧版评论区的适配删掉应该就是没问题的（不过好像对旧版评论区的 UP 标识的适配没了，不知道是不是之前适配新版评论区的时候改的）。
![image](https://user-images.githubusercontent.com/49396421/139567880-f05057b3-d865-4ae0-aadf-436584351881.png)

至于其他杂七杂八的小脚本，我还是建议让脚本自己适配 Bilibili Evolved 的夜间模式（有脚本做过这事，如：[[Bilibili] 关注管理器](https://greasyfork.org/scripts/428895)）。

或者像我在 https://github.com/the1812/Bilibili-Evolved/issues/1146#issuecomment-720884825 提过的，参考 [GitHub Dark](https://github.com/StylishThemes/GitHub-Dark)，独立出一个夜间模式的 repo，把适配工作交给开源社区，并同时适配市面上各种使用人数比较多的插件／脚本。
